### PR TITLE
Add 1ptbar index and modify soulvoice login to cookie

### PR DESCRIPTION
**Pull request by reprover**
I'm coming again!

### CODE REVIEW

File: src/Jackett.Common/Definitions/1ptbar.yml [hunk 1]
```diff
@@ -0,0 +1,157 @@
+---
+id: 1ptbar
+name: 1ptbar
+description: "SoulVoice is a CHINESE Private Torrent Tracker for Movies, TV, and e-Learning"
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/1ptbar.yml:4]: you forgot to update the description which is still set to soulvoice

File: src/Jackett.Common/Definitions/1ptbar.yml [hunk 2]
```diff
@@ -0,0 +1,157 @@
+---
+id: 1ptbar
+name: 1ptbar
```
Comment (garfield69) [inline src/Jackett.Common/Definitions/1ptbar.yml:3]: don't forget to add this name to the readme.md in the private list section

**Review by garfield69 — CHANGES_REQUESTED** [review]
In future, when adding a new site, please raise a separate PR
in the event we have to back out your soulvoice change, we end up having to also back out 1ptbar.
having separate PR allows better management of changes.

**Comment by reprover** [comment]
I'm sorry for my error. I'll create a new branch tomorrow since there's very late in my location. 
btw,  I'm sure it works well beacuse I'm using it :-)

**Review by garfield69 — APPROVED** [review]

**Comment by garfield69** [comment]
Jackett 0.16.1327

**Pull request merged**